packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1447,6 +1449,9 @@ export default class TradeStore extends BaseStore {
             this.prev_chart_layout.is_used = false;
         }
         this.resetAccumulatorData();
+        if (this.is_vanilla) {
+            this.setBarrierChoices([]);
+        }
```

Comment:
what is this for?

Reply:
When the page is reloaded I want to update `barrier_choices` (`strike_choices`) in order to set default value as chosen. If I won't do this, after user reloaded the page he will still see the value, that he chosen before (not default)

Reply:
this is not applicable to the scope of the card right ? just an additional bug fix?

Reply:
In previous Vanilla implementation there were no such problem, but after it was rewritten the behaviour changed, So in order to keep the old behaviour I add this

Reply:
mmm weird wonder what changed

Reply:
In _packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.js_ I started to keep `strike_prices` array with chosen one in order to store this info (while switching between Call/Put and etc.), so after user reload the page his choice is still in localStorage) That is why I need to clean it